docs/content/setup/manual-setup.md

```diff
@@ -1,41 +1,137 @@
-3. Enter the directory and type `bin/setup`, which will install the dependencies and create configs.
-4. Modify the file named `config.json` or configure HedgeDoc through environment variables which will overwrite the configs, see docs [here](../configuration.md).
-5. **If using the release tarball for 1.7.0 or newer, this step can be skipped.**  
-   Build the frontend bundle by `yarn run build` (use `yarn run dev` if you are in development)
-6. It is recommended to start your server manually once: `NODE_ENV=production yarn start`, this way it's easier to see warnings or errors that might occur (leave out `NODE_ENV=production` for development).
-7. Run the server as you like (node, forever, pm2, SystemD, Init-Scripts)
-
-## How to upgrade your installation
+!!! info "Requirements on your server"
+    - Node.js 10.13 or higher
+    - Database (PostgreSQL, MySQL, MariaDB, SQLite, MSSQL)  
+      The database must use charset `utf8`. This is typically the default in PostgreSQL and SQLite.  
+      In MySQL and MariaDB UTF-8 might need to be set with `alter database <DBNAME> character set utf8 collate utf8_bin;`  
+      Be aware of older MySQL and MariaDB versions which sometimes use shorter representations of UTF-8 than 4 bytes.
+      This can break if symbols with more bytes are used.
+      You can use `alter database <DBNAME> character set utf8mb4 COLLATE utf8mb4_unicode_ci` to be on the safe side.
+    - NPM (and its dependencies, [node-gyp](https://github.com/nodejs/node-gyp#installation))
+    - [Yarn Classic](https://classic.yarnpkg.com) 1.22 or higher (Yarn 2 is currently not supported)
+    - Bash (for the setup script)
+    - For **building** the HedgeDoc frontend you need a machine with at least **2 GB** RAM.
+      - Starting with release 1.7 the release tarball includes the prebuilt frontend, so building it yourself is not necessary.
+
+1. Check if you meet the [requirements at the top of this document](#manual-installation).
+2. Download the [latest release](https://hedgedoc.org/latest-release/) and extract it.  
+   <small>Alternatively, you can use Git to clone the repository and checkout a release, e.g. with `git clone -b 1.7.2 https://github.com/hedgedoc/hedgedoc.git`.</small>
+3. Enter the directory and type `bin/setup`, which will install the dependencies and create example configs.
+4. Configure HedgeDoc: To get started, you can use this minimal `config.json`:
+   ```json
+   {
+     "production": {
+       "db": {
+         "dialect": "sqlite",
+         "storage": "./db.hedgedoc.sqlite"
+      },
+       "urlAddPort": true,
+       "domain": "localhost"
+     }
+   }
+   ```
+   It's also possible to use environment variables.
+   For details, have a look at [the configuration documentation](../configuration.md).
+5. *:octicons-light-bulb-16: If you used the release tarball for 1.7.0 or newer, this step can be skipped.*  
```

Comment:
```suggestion
5. *:octicons-light-bulb-16: If you use the release tarball for 1.7.0 or newer, this step can be skipped.*  
```

Reply:
IMHO if you use present tense in every other bullet point and you should use it everywhere. :shrug: